src/applications/personalization/profile/tests/e2e/contact-information/confirm-email-address-no-email.cypress.spec.js

```diff
@@ -0,0 +1,116 @@
+import { PROFILE_PATHS } from '@@profile/constants';
+import mockUser from '@@profile/tests/fixtures/users/user-36.json';
+import {
+  mockFeatureToggles,
+  mockGETEndpoints,
+} from '@@profile/tests/e2e/helpers';
+
+const MHV_EMAIL_CONFIRMATION_DISMISSED_COOKIE =
+  'MHV_EMAIL_CONFIRMATION_DISMISSED';
+
+const setup = () => {
+  mockFeatureToggles(() => ({
+    data: {
+      type: 'feature_toggles',
+      features: [
+        {
+          name: 'mhvEmailConfirmation',
+          value: true,
+        },
+      ],
+    },
+  }));
+
+  const mockUserNoEmail = { ...mockUser };
+  mockUserNoEmail.data.attributes.vet360ContactInformation.email.emailAddress = null;
+  cy.login(mockUserNoEmail);
+
+  mockGETEndpoints([
+    'v0/mhv_account',
+    'v0/profile/full_name',
+    'v0/profile/status',
+    'v0/profile/personal_information',
+    'v0/profile/service_history',
+    'v0/profile/direct_deposits',
+  ]);
+
+  cy.intercept('GET', '/v0/user?*', {
+    statusCode: 200,
+    body: mockUserNoEmail,
+  }).as('finalUserRequest');
+
+  cy.visit(PROFILE_PATHS.CONTACT_INFORMATION);
+  cy.wait('@mockUser');
+};
+
+const clickAddEmailAddress = () => {
```

Comment:
These functions should go in the page file `src/applications/mhv-landing-page/tests/e2e/pages/LandingPage.js`. This is pattern the VA wants. https://depo-platform-documentation.scrollhelp.site/developer-docs/best-practices-for-using-cypress#BestpracticesforusingCypress-PageObjects